db/migrate/20240918233930_add_fetched_replies_at_to_status.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddFetchedRepliesAtToStatus < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
```

Comment:
You don't need this in this migration.

Reply:
thx, one of those cargo cult code lines i just add by reflex because i'm not exactly sure what it does lol <3